bundles/org.openhab.binding.openwebnet/src/main/java/org/openhab/binding/openwebnet/handler/OpenWebNetThermoregulationHandler.java

```diff
@@ -0,0 +1,375 @@
+        WhereThermo wt = new WhereThermo(wStr);
+        if (wt.isProbe()) {
+            isTempSensor = true;
+        }
+        return wt;
+    }
+
+    @Override
+    protected String ownIdPrefix() {
+        return Who.THERMOREGULATION.value().toString();
+    }
+
+    private void handleSetFanSpeedCommand(Command command) {
+        logger.debug("handleSetFanSpeedCommand() (command={})", command);
+        if (command instanceof StringType) {
+            Where w = deviceWhere;
+            if (w != null) {
+                try {
+                    FAN_COIL_SPEED speed = FAN_COIL_SPEED.valueOf(command.toString());
+                    send(Thermoregulation.requestWriteFanCoilSpeed(w.value(), speed));
+                } catch (OWNException e) {
+                    logger.warn("handleSetFanSpeedCommand() {}", e.getMessage());
+                } catch (IllegalArgumentException e) {
+                    logger.warn("handleSetFanSpeedCommand() Unsupported command {} for thing {}", command,
+                            getThing().getUID());
+                    return;
+                }
+            }
+        } else {
+            logger.warn("handleSetFanSpeedCommand() Unsupported command {} for thing {}", command, getThing().getUID());
+        }
+    }
+
+    private void handleSetpointCommand(Command command) {
+        logger.debug("handleSetpointCommand() (command={})", command);
+        if (command instanceof QuantityType || command instanceof DecimalType) {
+            Where w = deviceWhere;
+            if (w != null) {
+                BigDecimal value = BigDecimal.ZERO;
+                if (command instanceof QuantityType) {
+                    Unit<Temperature> unit = CELSIUS;
+                    QuantityType<Temperature> quantity = commandToQuantityType(command, unit);
+                    value = quantity.toBigDecimal();
+                } else {
+                    value = ((DecimalType) command).toBigDecimal();
+                }
+                try {
+                    send(Thermoregulation.requestWriteSetpointTemperature(w.value(), value.floatValue(),
+                            currentFunction));
+                } catch (MalformedFrameException | OWNException e) {
+                    logger.warn("handleSetpointCommand() {}", e.getMessage());
+                }
+            }
+        } else {
+            logger.warn("handleSetpointCommand() Unsupported command {} for thing {}", command, getThing().getUID());
+        }
+    }
+
+    private void handleMode(Command command) {
+        if (command instanceof StringType) {
+            Where w = deviceWhere;
+            if (w != null) {
+                try {
+                    OPERATION_MODE mode = OPERATION_MODE.valueOf(((StringType) command).toString());
+                    if (mode == OPERATION_MODE.MANUAL) {
+                        logger.debug("handleMode() mode={} function={} setPointTemp={}°", mode.toString(),
```

Comment:
Same for the others.
```suggestion
                        logger.debug("handleMode() mode={} function={} setPointTemp={}°", mode,
```

Reply:
removed entire line